tutorial/001_first.py

```diff
@@ -0,0 +1,115 @@
+"""
+.. _first:
+
+First Optimization
+==================
+
+Quadratic Function Example
+--------------------------
+
+Usually, Optuna is used to optimize hyper-parameters, but as an example,
+let us directly optimize a quadratic function in an IPython shell.
+"""
+
+
+import optuna
+
+###################################################################################################
+# The objective function is what will be optimized.
+
+
+def objective(trial):
+    x = trial.suggest_uniform('x', -10, 10)
+    return (x - 2) ** 2
+
+
+###################################################################################################
+# This function returns the value of :math:`(x - 2)^2`. Our goal is to find the value of ``x``
+# that minimizes the output of the ``objective`` function. This is the "optimization."
+# During the optimization, Optuna repeatedly calls and evaluates the objective function with
+# different values of ``x``.
+#
+# A :class:`~optuna.trial.Trial` object corresponds to a single execution of the objective
+# function and is internally instantiated upon each invocation of the function.
+#
+# The `suggest` APIs (for example, :func:`~optuna.trial.Trial.suggest_float`) are called
+# inside the objective function to obtain parameters for a trial.
+# :func:`~optuna.trial.Trial.suggest_float` selects parameters uniformly within the range
+# provided. In our example, from :math:`-10` to :math:`10`.
+#
+# To start the optimization, we create a study object and pass the objective function to method
+# :func:`~optuna.study.Study.optimize` as follows.
+
+study = optuna.create_study()
+study.optimize(objective, n_trials=100)
+
+
+###################################################################################################
+# You can get the best parameter as follows.
+
+print(study.best_params)
+
+###################################################################################################
+# We can see that Optuna found the best ``x`` value is close to the optimal value of ``2``.
```

Comment:
I'm wondering if users will be confused by this sentence, since the meanings of "best" and "optimal" are very close. Therefore, I think we should say that the best value "found by Optuna" is close enough to the optimal value.